requirements.txt

```diff
@@ -0,0 +1,5 @@
+pylint
+flake8
+nose
```

Comment:
why do we need nose and jedi?

Reply:
doesn't look like it's imported anywhere

Reply:
These are just convenient dev tools which are usually a part of the dev env:
1. Jedi powers auto completion, goto and refactoring.
2. Nose is a better runner for `unittest` which supports stdout redirection, pdb, etc.

I can do one of the following:
1. Rename the `requirements.txt` to `requirements-dev.txt` to indicate that this file concerns dev env only.
2. Create a separate `requirements-dev.txt` and move jedi and nose there.
3. Just drop jedi and nose from the `requirements.txt`.
I'm ok with either approach.

Reply:
let's go with 3 because they are not required, i don't use jedi for dev env, my vim already has syntax highliting and auto completion. you can always install jedi and nose locally

the only real requirements for the probject seem to be pylint and black ongoing

Reply:
Sounds good. Done that